code/modules/mob/dead/new_player/login.dm

```diff
@@ -32,10 +32,10 @@
 	new_player_panel()
 	client.playtitlemusic()
 	if(SSticker.current_state < GAME_STATE_SETTING_UP)
-		var/tl = round(SSticker.GetTimeLeft(), 1)/10
+		var/tl = SSticker.GetTimeLeft()
 		var/postfix
-		if(tl >= 0)

```

Comment:
it's set to -1, iirc, to delay, so you might need to account for that

Reply:
Honk. I actually had set it to > in the first place, but looked at it again a few minutes later and forgot all about it, haha.

Reply:
yea, negative numbers here will delay